### ISSUE TITLE
Import der Spiele verbessern

### ISSUE THREAD

**Issue by HendrikD** [HendrikD/obiblio#4]

**Comment by LeAchim59** [comment]
Der Import von Daten ist nicht Bestandteil dieses Softwarepakets, sondern eigene Programme, die sich direkt an der Datenbank anmelden und CSV-Datein in die Tabellen Biblio und BiblioCopy

**Comment by HendrikD** [comment]
Okay. Was sind das für Programme? 

Ich spinne mal ein bisschen rum: Ich hatte gedacht es wäre eigentlich cool wenn man einfach die Barcodes von den Spielen scannen könnte und dann in einer offenen EAN-Datenbank (opengtindb.org)  nach dem Spiel sucht. Dann haben wir Namen und Hersteller vom Spiel. Dann können wir prüfen ob wir das Spiel schon im Bestand haben. Wenn ja -> Neues Exemplar hinzufügen. Wenn nein -> Über die Boardgamegeek-API die Metadaten zu dem Spiel laden und Spiel im System anlegen. Dann Exemplar hinzufügen.

Bei Hinzufügen des Exemplars könnte man dann noch einen Barcode/QR-Code mit der Exemplarnummer generieren den man dann auf einem Stickersheet ausdruckt, auf das Spiel klebt und für eine schnelle Ausleihe und Rückgabe scannen kann.

Könnte man dann überlegen einen Handscanner anzuschaffen https://amzn.to/3a88mUe

**Comment by LeAchim59** [comment]
Ja, ist aber leider nicht machbar, da wir die EAN der Spiele immer mit unseren Barcodes überkleben, damit es bei Darmstadt spielt nicht zu Fehllesungen kommt.
Wir können keine Barcodes generieren, da sie auch für Darmstadt spielt funktionieren müssen. Bzw. müssten das berücksichtigen. Nie bei Darmstadt spielt an der Ausleihe geholfen? Alle Spiele haben bereits eindeutige Barcodes. 
So eine Funktion wäre also eher was für die Zukunft für Neuanschaffungen. Den Bestand werden wir importieren müssen.

Das sind kleine php-Programme, die eine CSV-Datei lesen und in die Datenbank importieren. Das ist jetzt nicht unbedingt erforderlich

**Comment by HendrikD** [comment]
Nein ich habe bei Darmstadt spielt noch nicht an der Ausleihe geholfen. Diese eindeutigen Barcodes sollten wir dann vermutlich auch für die Exemplatidentifizierung nutzen.

**Comment by HendrikD** [comment]
Müssen wir hier jetzt irgendwas an openbiblio ändern? So wie ich das verstehe nicht. Ist eine separate Software.

**Comment by LeAchim59** [comment]
Ja, mit meinem Importprogramm komme ich zurecht, das lassen wir erst mal so.
